### PR TITLE
Hybrid WHfB: MarkDown code format, whitespace, typos

### PR DESCRIPTION
**Pull request by KarkDenUheldige**
**Description:**

As mentioned in PR #8595, I noticed quite a few formatting issues in this document, both for MarkDown code block formatting and whitespace usage (or lack thereof).

Since the PR was focused on a quite specific topic update, the PR author did not want additional changes added. I therefore submit a list of my own improvements.

**Changes proposed:**

- add MarkDown code block syntax highlighting language `aspx` for the schema scripts (might be asp.net or XML)
- reformat and wrap the example script for removing any existing issuerID claim (Remarks section)
- add proper MarkDown code blocks around separate command lines and important lists
- add PowerShell syntax highlighting to PS MarkDown code blocks
- remove PowerShell command prompt for command copying (current codestyle)
- remove redundant ending ### in the heading "#### ADFS Web Proxy ###"
- typo correction: replace "and" with "a" in "open and command prompt"
- add missing colon at the end of introductory sentences

**Whitespace changes:**

- add missing blank lines between section headings (H2/H3/H4) and their text paragraphs
- remove any blank space at the end of lines (whitespace cleanup)
- standardize number list spacing to a single blank (MarkDown standard)

**Ticket closure or reference:**

Ref. #8595

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
Due to the high amount of whitespace changes in this PR, I recommend using the feature **Hide whitespace changes:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8686/files?diff=unified&w=1 (added blank lines are not skipped, but most of my removal of simple blanks is hidden in this view).

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
@e0i : For Your Information & please assign Matthew (mapalko) for code review. Request copy review if you find it practical.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 1]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+> [!IMPORTANT]
 > To use hybrid identity with Azure Active Directory and device WriteBack features, you must use the built-in GUI with the [latest updates for ADConnect](https://www.microsoft.com/download/details.aspx?id=47594).
 
 ## Configure Azure for Device Registration
-Begin configuring device registration to support Hybrid Windows Hello for Business by configuring device registration capabilities in Azure AD. 
 
-To do this, follow the **Configure device settings** steps under [Setting up Azure AD Join in your organization](https://azure.microsoft.com/documentation/articles/active-directory-azureadjoin-setup/)  
+Begin configuring device registration to support Hybrid Windows Hello for Business by configuring device registration capabilities in Azure AD.
+
+To do this, follow the **Configure device settings** steps under [Setting up Azure AD Join in your organization](https://azure.microsoft.com/documentation/articles/active-directory-azureadjoin-setup/)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:57]:
```suggestion
To do this, follow the **Configure device settings** steps under [Setting up Azure AD Join in your organization](https://azure.microsoft.com/documentation/articles/active-directory-azureadjoin-setup/).
```

Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:57]:
```suggestion
To do this, follow the **Configure device settings** steps under [Setting up Azure AD Join in your organization](https://azure.microsoft.com/documentation/articles/active-directory-azureadjoin-setup/).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 2]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
-Windows Hello for Business uses asymmetric keys as user credentials (rather than passwords).  During enrollment, the public key is registered in an attribute on the user object in Active Directory.  The schema update adds this new attribute to Active Directory.  
+Windows Hello for Business uses asymmetric keys as user credentials (rather than passwords). During enrollment, the public key is registered in an attribute on the user object in Active Directory. The schema update adds this new attribute to Active Directory.
 
-Manually updating Active Directory uses the command-line utility **adprep.exe** located at **\<drive>:\support\adprep** on the Windows Server 2016 or later DVD or ISO.  Before running adprep.exe, you must identify the domain controller hosting the schema master role.
+Manually updating Active Directory uses the command-line utility **adprep.exe** located at **\<drive>:\support\adprep** on the Windows Server 2016 or later DVD or ISO. Before running adprep.exe, you must identify the domain controller hosting the schema master role.
 
 Sign-in to the domain controller hosting the schema master operational role using enterprise administrator equivalent credentials.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:88]:
```suggestion
Sign in to the domain controller hosting the schema master operational role using enterprise administrator equivalent credentials.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 3]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+2. Type `cd /d x:\support\adprep` where *x* is the drive letter of the DVD or mounted ISO.
+3. To update the schema, type `adprep /forestprep` .
+4. Read the Adprep Warning. Type the letter **C*** and press **Enter** to update the schema.
+5. Close the Command Prompt and sign out.
 
 > [!NOTE]
 > If you installed Azure AD Connect prior to upgrading the schema, you will need to re-run the Azure AD Connect installation and refresh the on-premises AD schema to ensure the synchronization rule for msDS-KeyCredentialLink is configured.
 
 
 ### Setup Active Directory Federation Services
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:100]:
```suggestion
### Set up Active Directory Federation Services
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 4]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+
 If you are new to AD FS and federation services, you should review [Understanding Key AD FS Concepts](https://docs.microsoft.com/windows-server/identity/ad-fs/technical-reference/understanding-key-ad-fs-concepts) to prior to designing and deploying your federation service.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:102]:
```suggestion
If you are new to AD FS and federation services, you should review [Understanding Key AD FS Concepts](https://docs.microsoft.com/windows-server/identity/ad-fs/technical-reference/understanding-key-ad-fs-concepts) prior to designing and deploying your federation service.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 5]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
 
 > [!NOTE]
-> If necessary, copy the AdSyncPrep.psm1 file from your Azure AD Connect server.  This file is located in Program Files\Microsoft Azure Active Directory Connect\AdPrep
+> If necessary, copy the AdSyncPrep.psm1 file from your Azure AD Connect server. This file is located in Program Files\Microsoft Azure Active Directory Connect\AdPrep .
 
-![Device Registration](images/hybridct/device6.png)   
+![Device Registration](images/hybridct/device6.png)
 
-2. Provide your Azure AD global administrator credentials  
+2. Provide your Azure AD global administrator credentials
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:177]:
```suggestion
2. Provide your Azure AD global administrator credentials.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 6]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
 
-1.  Open Windows PowerShell and execute the following:  
+1. Open Windows PowerShell and execute the following:
+
+   ```PowerShell
+   Initialize-ADSyncDeviceWriteBack -DomainName <AD DS domain name> -AdConnectorAccount [AD connector account name]
+   ```
 
-    `PS C:>Initialize-ADSyncDeviceWriteBack -DomainName <AD DS domain name> -AdConnectorAccount [AD connector account name]` 
+Where the [AD connector account name] is the name of the account you configured in Azure AD Connect when adding your on-premises AD DS directory in domain\accountname format
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:205]:
```suggestion
Where the [AD connector account name] is the name of the account you configured in Azure AD Connect when adding your on-premises AD DS directory in domain\accountname format.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 7]
```diff
@@ -463,64 +502,77 @@ The following script helps you with the creation of the issuance transform rules
...
-For your reference, below is a comprehensive list of the AD DS devices, containers and permissions required for device write-back and authentication to work
+```PowerShell
+Set-AdfsGlobalAuthenticationPolicy -DeviceAuthenticationEnabled $true -DeviceAuthenticationMethod SignedToken
+```
 
-- object of type ms-DS-DeviceContainer at CN=RegisteredDevices,DC=&lt;domain&gt;        
-    - read access to the AD FS service account   
+#### Check your configuration
+
+For your reference, below is a comprehensive list of the AD DS devices, containers and permissions required for device write-back and authentication to work:
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:564]:
```suggestion
For your reference, below is a comprehensive list of the AD DS devices, containers, and permissions required for device write-back and authentication to work:
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 8]
```diff
@@ -463,64 +502,77 @@ The following script helps you with the creation of the issuance transform rules
...
+    ```
 
 - If you have already issued an **ImmutableID** claim  for user accounts, set the value of **$immutableIDAlreadyIssuedforUsers** in the script to **$true**.
 
-#### Configure Device Authentication in AD FS  
-Using an elevated PowerShell command window, configure AD FS policy by executing the following command  
+#### Configure Device Authentication in AD FS
 
-`PS C:>Set-AdfsGlobalAuthenticationPolicy -DeviceAuthenticationEnabled $true -DeviceAuthenticationMethod SignedToken` 
+Using an elevated PowerShell command window, configure AD FS policy by executing the following command
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:556]:
```suggestion
Using an elevated PowerShell command window, configure AD FS policy by executing the following command:
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 9]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+> During your AD FS deployment, skip the **Configure a federation server with Device Registration Service** and the **Configure Corporate DNS for the Federation Service and DRS** procedures.
 
-The AD FS farm used with Windows Hello for Business must be Windows Server 2016 with minimum update of [KB4088889 (14393.2155)](https://support.microsoft.com/help/4088889).  If your AD FS farm is not running the AD FS role with updates from Windows Server 2016, then read [Upgrading to AD FS in Windows Server 2016](https://docs.microsoft.com/windows-server/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server-2016)
+The AD FS farm used with Windows Hello for Business must be Windows Server 2016 with minimum update of [KB4088889 (14393.2155)](https://support.microsoft.com/help/4088889). If your AD FS farm is not running the AD FS role with updates from Windows Server 2016, then read [Upgrading to AD FS in Windows Server 2016](https://docs.microsoft.com/windows-server/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server-2016)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:110]:
```suggestion
The AD FS farm used with Windows Hello for Business must be Windows Server 2016 with minimum update of [KB4088889 (14393.2155)](https://support.microsoft.com/help/4088889). If your AD FS farm is not running the AD FS role with updates from Windows Server 2016, then read [Upgrading to AD FS in Windows Server 2016](https://docs.microsoft.com/windows-server/identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server-2016).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 10]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+   Import-module activedirectory
+   Initialize-ADDeviceRegistration -ServiceAccountName "<your service account>"
+   ```
 
-   `Import-module activedirectory`  
-   `PS C:\> Initialize-ADDeviceRegistration -ServiceAccountName "<your service account>"` 
 3. On the pop-up window click **Yes**.
 
 > [!NOTE]
 > If your AD FS service is configured to use a GMSA account, enter the account name in the format "domain\accountname$"
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:146]:
```suggestion
> If your AD FS service is configured to use a GMSA account, enter the account name in the format "domain\accountname$".
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 5 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:177]:
```suggestion
2. Provide your Azure AD global administrator credentials.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 11]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
-    `PS C:>$aadAdminCred = Get-Credential`
+   ```PowerShell
+   $aadAdminCred = Get-Credential
+   ```
 
-![Device Registration](images/hybridct/device7.png) 
+![Device Registration](images/hybridct/device7.png)
 
-3. Run the following PowerShell command 
+3. Run the following PowerShell command
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:185]:
```suggestion
3. Run the following PowerShell command:
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 6 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:205]:
```suggestion
Where the [AD connector account name] is the name of the account you configured in Azure AD Connect when adding your on-premises AD DS directory in domain\accountname format.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 12]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+
+   ```PowerShell
+   Initialize-ADSyncDeviceWriteBack -DomainName <AD DS domain name> -AdConnectorAccount [AD connector account name]
+   ```
 
-    `PS C:>Initialize-ADSyncDeviceWriteBack -DomainName <AD DS domain name> -AdConnectorAccount [AD connector account name]` 
+Where the [AD connector account name] is the name of the account you configured in Azure AD Connect when adding your on-premises AD DS directory in domain\accountname format
 
-Where the [AD connector account name] is the name of the account you configured in Azure AD Connect when adding your on-premises AD DS directory in domain\accountname format  
+The above command creates the following objects for device write back to AD DS, if they do not exist already, and allows access to the specified AD connector account name
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:207]:
```suggestion
The above command creates the following objects for device write back to AD DS, if they do not exist already, and allows access to the specified AD connector account name:
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 13]
```diff
@@ -45,148 +47,171 @@ Use this three-phased approach for configuring device registration.
...
+- RegisteredDevices container in the AD domain partition
+- Device Registration Service container and object under Configuration --> Services --> Device Registration Configuration
 
-- RegisteredDevices container in the AD domain partition  
-- Device Registration Service container and object under Configuration --> Services --> Device Registration Configuration  
+### Enable Device Write Back in Azure AD Connect
 
-### Enable Device Write Back in Azure AD Connect  
-If you have not done so before, enable device write back in Azure AD Connect by running the wizard a second time and selecting **"Customize Synchronization Options"**, then checking the box for device write back and selecting the forest in which you have run the above cmdlets  
+If you have not done so before, enable device write back in Azure AD Connect by running the wizard a second time and selecting **"Customize Synchronization Options"**, then checking the box for device write back and selecting the forest in which you have run the above cmdlets
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:214]:
```suggestion
If you have not done so before, enable device write back in Azure AD Connect by running the wizard a second time and selecting **"Customize Synchronization Options"**, then checking the box for device write back and selecting the forest in which you have run the above cmdlets.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor additional edits, please.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 1 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:57]: Good point. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 2 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:88]: Well spotted. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 3 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md]: Good job. Well spotted.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 4 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:102]: Nice. Well spotted. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 7 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:564]: Thank you. Fair point.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 8 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:556]: Well spotted. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 9 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:110]: Good point. Well spotted.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 10 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:146]: Good point. Well spotted.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 5 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:177]: Fair point. Good decision.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 11 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:185]: Nice. Well spotted.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 6 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:205]: Good point. Very well spotted. It feels like that sentence should have been enclosed in parentheses and be a finalizing item in an already existing sentence before the code example.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md (hunk 13 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md:214]: Well spotted. Thank you.

**Comment by illfated** [comment]
@JohanFreelancer9 : Very good job. Well spotted. Thank you for pointing out all those missing punctuation items in this document. Feels almost like I did not make much of an effort to check the document for proper punctuation.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @illfated - looks good.

**Comment by illfated** [comment]
Thank **you**. Your confirmation is very welcome.

**Comment by illfated** [comment]
I recommend using **Hide whitespace changes** to reduce the amount of lines for review, but more importantly, to align the actual changes better:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8686/files?diff=unified&w=1 

(Some of the whitespace changes are causing disorder in the appearance of factual changes, making it look like I have altered some text content instead of only typo corrections.)

**Comment by illfated** [comment]
Sorry for the disruption. I made some mistakes during branch clean-up in my repo.

**Comment by illfated** [comment]
@e0i : May I ask you to please check if this PR has been dislodged from any automatic listing for PRs to be followed up, in case my unwanted closure and re-opening caused a technical problem?

**Pull request closed without merging**